src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/TestSignatures/11.json

```diff
@@ -0,0 +1,66 @@
+{
+  "Signatures": [
+    {
+      "Label": "Boolean(text)",
+      "Documentation": "Converts a 'text' that represents a boolean to a boolean value.",
+      "Parameters": [
+        {
+          "Label": "text",
+          "Documentation": "The text value to convert to a boolean value."
+        }
+      ]
+    },
+    {
+      "Label": "Boolean(input)",
+      "Documentation": "Converts a column of text values to a column of boolean values.",
+      "Parameters": [
+        {
+          "Label": "input",
+          "Documentation": ""
```

Comment:
>"" [](http://example.com/codeflow?start=27&length=2)

@MikeStall this highlights we are missing documentation for functions,
filed this to track: [https://github.com/microsoft/Power-Fx/issues/1975](https://github.com/microsoft/Power-Fx/issues/1975)

Reply:
Be careful when doing this. If this is assigned a different description then other versions of input for the same function and its overloads then we would need to factor in the description of the active parameter when comparing two signature information.

Reply:
I've left that out to avoid the complexity here.